spec/models/education_spec.rb

```diff
@@ -1,5 +1,56 @@
 require 'rails_helper'
 
 RSpec.describe Education, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  # it "should be a valid Education" do
+    # application = FactoryGirl.build(:application)
+    # education = Education.new(
+    #   application_id: application.id,
+    #   school: "Montessori Awesome School",
+    #   location: "The middle of Nowhere",
+    #   start_date: "2017-06-15 19:36:36",
+    #   end_date: "2017-06-10 19:36:36",
+    #   degree: "Bachelors of Many Things",
+    #   degree_date: "2017-06-11 19:36:36"
+    # )
+    # expect(education).to be_valid
+  # end
+  it "Education model should have a valid factory with faker" do
```

Comment:
I think the lines should read `it "is valid with all necessary attributes`. Where did you get the `should` from? That's actually deprecated for rspec. Check out http://www.betterspecs.org/#single to get an idea of what this should look like